servers/biometric_signal_sensor_interface/src/hexoskinHelper/resourceHelper.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import absolute_import, division, print_function
+import sys
+
+__author__ = 'abhijith'
+
+'''
+biometric resourceHelper provides all methods that retrieves the biometric
+data. This helper module is called by the Tango server module providing a clear
+line of abstraction.
+'''
+
+
+def getRecordListHelper(auth, user):
```

Comment:
Python coding style requires function's and method's names to be snake_case, and class names to be CamelCase